easybuild/easyblocks/t/tensorflow.py

```diff
@@ -81,8 +81,13 @@ def extra_options():
             'cuda_compute_capabilities': [[], "List of CUDA compute capabilities to build with", CUSTOM],
             'path_filter': [[], "List of patterns to be filtered out in paths in $CPATH and $LIBRARY_PATH", CUSTOM],
             'with_jemalloc': [None, "Make TensorFlow use jemalloc (usually enabled by default)", CUSTOM],
-            'with_mkl_dnn': [True, "Make TensorFlow use Intel MKL-DNN", CUSTOM],
         }
+
```

Comment:
This makes sense to me, but I think we can simultaneously tackle a small issue. Down in the easyblock we enable `mkl-dnn` https://github.com/easybuilders/easybuild-easyblocks/blob/98984e2e6393c09642b4111b9f97a490007f857e/easybuild/easyblocks/t/tensorflow.py#L382 However, that line doesn't depend on CUDA. So it is actually an error, since you can't have simultaneously a tensorflow build with `mkl-dnn` and `cuDNN`. Or can you? @akesandgren ? Does it make sense to make that block an `else:` of `if cuda_root:`?

Reply:
I've gone for "if cudnn is present then don't use mkl-dnn"... does that look ok?